packages/db-migrations/databases/fxa/patches/patch-163-164.sql

```diff
@@ -0,0 +1,18 @@
+-- Remove `email` column from the `carts` table.
+-- `email` field can be populated via the `uid` column in the `accounts` table.
+-- Throw an error if any carts have an email value but no uid value. Only update the patch level if successful
```

Comment:
Why is this necessary? Could the column just be dropped?

Reply:
I'm inclined to say it's not worth it. But maybe I'm not seeing risk here.

Assume, we hard drop the email column, regardless of if there was or wasn't a uid. What would be the risk of having a cart row which previously had an email, but doesn't have a uid?

Reply:
the main concern around this is that we may have carts that were previously attributed to a stub account via their email, but that didn't have an actual uid associated with them. Do you know if that's likely to be an issue? I don't recall if the stub account logic was ever actually live